### ISSUE TITLE
On (probable) discontinuation of erigon support for BSC

### ISSUE THREAD

**Issue by 0xabhinav** [allada/bsc-archive-snapshot#51]
Hi @allada! Thanks for maintaining erigon snapshots for BSC! Your scripts and data have been highly useful to us in setting up our own BSC nodes

Unfortunately, as per [this article](https://erigon.substack.com/p/future-of-the-support-of-bnb-smart) I came across recently, it seems erigon is soon about to drop support for BSC from their node. Wonder what's the right call here.. is it possible for you to update the script to go back to using geth archive nodes?

Thanks again for your community effort! Cheers

**Comment by allada** [comment]
Thanks for brining this to my attention. I do not have the time and resources right now to maintain Erigon for BSC. Given this, if Erigon drops support for BCS it will likely mean the eventual freeze of this project.

As it stands right now, nearly every EVM paid service uses Erigon and at least has used this project, and only one has ever donated to this project and to my understanding none have donated to Erigon monetarily.

So in short, if they drop BSC I'll likely freeze this project and not maintain a go-ethereum version (as it would cost upwards of $4-8k per month in storage costs).

**Comment by 0xabhinav** [comment]
Understandable, thanks for your contribution!

**Comment by 0xabhinav** [comment]
Hi @allada!
As per erigon team, it seems like nodereal will be maintaining a fork for BSC at https://github.com/ledgerwatch/erigon/pull/7205#issuecomment-1494515160, just wanted to FYI so we can maintain this project post planck hardfork :)
